### PR TITLE
Feature/simple moe

### PR DESCRIPTION
**Pull request by hwchen**
- add simple moe agg, with basic formula of `sqrt(sum(power(moe / 1.645,
  2))) * 1.645`
- update previous moe to be called replicate weight moe.
- add critical value variable to replicate weight moe.
- add critical value variable to weighted avg moe.

### CODE REVIEW

**Comment by hwchen** [comment]
closes #209

File: tesseract-clickhouse/src/sql/aggregator.rs [hunk 1]
```diff
@@ -61,6 +61,12 @@ pub fn agg_sql_string_pass_1(col: &str, aggregator: &Aggregator, mea_idx: usize)
                 join(secondaries, ", "),
             )
         },
+        Aggregator::Moe { .. }=> {
+            format!("sum(power( {} / 1.645, 2)) as m{}_moe_sub_agg",
```
Comment (MarcioPorto) [inline tesseract-clickhouse/src/sql/aggregator.rs]: Maybe I am missing something, but should this `1.645` be substituted by the critical value?

File: tesseract-clickhouse/src/sql/aggregator.rs [hunk 2]
```diff
@@ -111,7 +117,7 @@ pub fn agg_sql_string_select_mea(aggregator: &Aggregator, mea_idx: usize) -> Str
                 mea_idx,
             )
         },
-        Aggregator::Moe { secondary_columns, .. }=> {
+        Aggregator::ReplicateWeightMoe { secondary_columns, .. }=> {
```
Comment (jspeis) [inline tesseract-clickhouse/src/sql/aggregator.rs]: should we add a space between the `}` and the  `=>`?

File: tesseract-clickhouse/src/sql/aggregator.rs (hunk 1 above)
Comment (hwchen) [reply tesseract-clickhouse/src/sql/aggregator.rs]: ah, yes, this one should be. thanks!

**Comment by hwchen** [comment]
Will document

**Comment by hwchen** [comment]
Added the beginnings of a docs folder. And there's examples for all the aggregators finally.

**Comment by hwchen** [comment]
@jspeis @MarcioPorto just checking if there's anything else before I merge. I added some docs for the aggregators. (and the beginnings of a docs folder, which could in the future be used for a static site generator)

**Comment by MarcioPorto** [comment]
Looks good to me!

**Comment by hwchen** [comment]
closes #209

**Pull request merged**